README.md

```diff
@@ -112,6 +112,8 @@ It results in the `ASWebAuthenticationSession` view not being closed properly, w
     }
     ```
     
+When implementing Swedish BankID authentication, the parent app must catch the redirect URI and prevent `handleAuthenticationResponse` from being called, to allow the authentication flow to finish. The URI scheme that is used to redirect back to the parent app from BankID will have the following format: `{app_uri_scheme}:/bankId`.
```

Comment:
Sounds good. What do you think about adding a special header for bankId like:
1. BankId
When implementing..... 

Also:
"the parent app must catch the redirect URI and returns with no action. Please make sure that `handleAuthenticationResponse` is not called for the BankID redirect" - sth like that sounds more clear to me. If you want you can also add a code snippet -> devs are lazy ;)